src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,115 @@
-export const Pagination = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+import './Pagination.scss';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number | string) => void;
+}
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const firstItem = (currentPage - 1) * perPage + 1;
+
+  const lastItem = firstItem + perPage > total
+    ? total
+    : firstItem + perPage - 1;
+  const recentItems = getNumbers(firstItem, lastItem).map((n: number) => `Item ${n}`);
```

Comment:
You calculate it twice. DRY - don't repeat yourself. Pass it as props and declare it once.

Reply:
@qoa11a I thought that I could use only these variables that was descripted in readme that is why I wrote this twice thought that it is not going to pass tests, but it is fine